src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1628,6 +1628,46 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
             $ref: '#/components/responses/defaultResponse'
+  /license/import:
+    post:
+      operationId: importLicense
+      tags:
+        - License
+      summary: Import a csv license
+      description: >
+        Import a csv license to the database
+      requestBody:
+        description: Information about delimiters, inclosure and csv file.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ImportLicenseRequest'
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/ImportLicenseRequest'
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                file_input:
+                  type: string
+                  format: binary
```

Comment:
Please note that content type can either be one of 3 provided and not a combination.
Thus, if someone is sending data as `application/json`, they can not send the `file_input` file.
Similarly, if someone is sending data as `multipart/form-data`, they can not send `delimiter` and `enclosure` as per the current schema.

Please merge them to `multipart/form-data` to allow file upload along with `delimiter` and `enclosure` parameters.

Reply:
great it makes sense.
What if I shift enclosure and delimiter properties in the headers?  🤔

Reply:
Done